docs/data-operations-manual/How-To-Guides/Adding/Add-a-new-dataset-and-collection.md

```diff
@@ -55,42 +51,67 @@ Follow steps 3 and 4 in ‘Adding an Endpoint’ for instructions on adding mapp
 
 Next, you can just proceed to add an endpoint. Validating the data through the endpoint checker will not work at this point but we can manually check whether the endpoint_url and documentation_url work as well as whether the required fields specified in the schema are present.   There will be no further need to validate it. Once the PR is merged, you can go to the next step.
 
-### 6. Test locally
+### 4. Test locally
+
+You can run a data collection pipeline locally and then verify your new configuration.  You can achieve this in two ways.
+
+#### 4.1 Run using config repo
 
-Clone the collection repo you created in step 2. Create a virtual environment with the following command
- python3 -m venv --prompt . .venv --clear --upgrade-deps
+The easiest option might be to just stay within the config repository.
+
+Create a virtual environment with the following command:
+```
+python3 -m venv --prompt . .venv --clear --upgrade-deps
+```
 
 You then need to update the collection with the dataset with the following commands:
-* make makerules
-* make init
-* make collect
-* make
+```
+make makerules
+make init
+make collect
+make
+```
+Finally. you should be able to run `make datasette` and then see a local datasette version with the new dataset on it.
+
+#### 4.2 Run using collection-task repo
+
+Clone the [collection-task](https://github.com/digital-land/collection-task) repo. 
+
+Set the environment variables to values relevant to the collection and dataset you have added within the docker-compose.yaml file. Then run with Docker Compose using `make compose-up`
 
+You should be able to run `make datasette` and then see a local datasette version with the new dataset on it.
 
-If everything is done correctly, you should be able to run `make datasette` and then see a local datasette version with the new dataset on it.
+#### Notes
 
-> Tip: When running make datasette and it returns that there is no sql file, make sure that the corresponding collection in config has an endpoint and source, else the database for it will not be set up!
+> Tip: When running make datasette and it returns that there is no SQL file, make sure that the corresponding collection in config has an endpoint and source, else the database for it will not be set up!
 
  > You’ll want to check the amount of records created in the dataset, this really should be equal to the amount of entries in the raw dataset. Also check the issues table for anything that sticks out and is fixable on our end
 
-### 7. Update run.yml and specification
+Once you're happy with the configuration for the new collection and new dataset, have it reviewed by a colleague and then merged.
+
+### 5. Regenerate Airflow DAGs
+
+Finally, you'll need to ensure the [DAGs for Airflow](https://github.com/digital-land/airflow-dags/) are regenerated and published to AWS.   This is necessary to ensure Airflow has the configuration need to run the new collection on a scheduled basis.
```

Comment:
How would someone do this? Does it require any configuration changes or does it just update based on the config repo changes? It would be good to have a bit more info on what to expect here.

Reply:
Thanks Greg, the instructions do follow this sentence but I've re-structured section to make it clearer.